### PR TITLE
Pr/dessygil/40

### PR DESCRIPTION
**Pull request by maclandrol**
Checklist:

- [x] Was this PR discussed in a issue? It is recommended to first discuss a new feature and let the community know whether you are planning or have started working on it before opening a PR.
- [x] Add tests to cover the fixed bug(s) or the new introduced feature(s) (if appropriate).
- [x] Update the API documentation if a new function is added or an existing one is deleted.
- [x] Added a `news` entry.
  - _copy `news/TEMPLATE.rst` to `news/my-feature-or-branch.rst`) and edit it._

---

xref #40  pinging @dessygil

### CODE REVIEW

**Comment by maclandrol** [comment]
Fix #43 

```python
from molfeat.trans.pretrained.hf_transformers import PretrainedHFTransformer
import datamol as dm

smiles = dm.freesolv()["smiles"]
transformer = PretrainedHFTransformer("ChemBERTa-77M-MLM", notation="smiles", precompute_cache=True)
output = PretrainedHFTransformer.batch_transform(transformer, smiles, batch_size=128, concatenate=False)
len(transformer.precompute_cache) # should be len(smiles). 
```

Pretrained models should now work better with `batch_transform`,  allowing efficient parallelization, while retaining all cached feature. `PrecomputedMolTransformer` molecule transformer should now be prefered WHEN you have existing cache already or are using static featurizers.

**Comment by maclandrol** [comment]
I have also documented missing featurizers now:

```python
from molfeat.store import ModelStore
store = ModelStore()
_, m = store.load("mordred")
print(m.usage())
```
```
import datamol as dm
from molfeat.trans import MoleculeTransformer
smiles = dm.freesolv().iloc[:50].smiles
# sanitize and standardize your molecules if needed
transformer = MoleculeTransformer(featurizer='mordred', dtype=float)
features = transformer(smiles)
```

ping @cwognum

File: plugin.yml
Comment (cwognum) [inline plugin.yml:1]: Not blocking, but any idea on how we could present this in a more aesthetically pleasing way? Maybe have them on the Molfeat website?

**Review by cwognum — APPROVED** [review]
LGTM!

File: plugin.yml
Comment (maclandrol) [reply plugin.yml:1]: Yes, I will ping @mercuryseries for that.

**Pull request merged**